internal/commands/commands.go

```diff
@@ -191,6 +191,10 @@ var (
 				Command:     &accesslist.CommandCreate{},
 				CommandMeta: accesslist.CommandMetaCreate,
 			},
+			{
```

Comment:
cc: @asudhakal ... this was missed in the [previous commit](https://github.com/10gen/realm-cli/commit/e7b26ce2292ced23a1bd7677771dea9df16e65e4) ... feel free to drive-by fix it in your current PR and i'll revert it from here, otherwise I've got it covered

Reply:
I got it for the current PR!